modules/tutorials-for-data-scientists.adoc

```diff
@@ -86,8 +86,8 @@ ifndef::upstream[]
 | Installing and verifying the NVIDIA GPU Add-on
```

Comment:
Not related to your actual change, but the NVIDIA GPU Add-on is no longer supported with RHODS. If this tutorial is still present in the product, it is a bug, and also needs removing from the documentation. Perhaps investigate as part of a new Jira/PR?

Reply:
Only the NVIDIA GPU **Operator** is supported by RHODS now.

Reply:
Thanks @chtyler ! Could you create a new ticket to correct the NVIDIA GPU details? Sounds like they appear in several places and it would be good to check the full project.